packages/server/src/routes/calendar/postEvent.ts

```
@@ -0,0 +1,93 @@
+import { createRoute, z } from "@hono/zod-openapi";
+import { InsertEventSchema, SelectEventSchema } from "../../drizzle/schema";
+import { authMiddleware } from "../middleware/authentication";
+import { EventTable } from "./event.dto";
+import { env } from "hono/adapter";
+import { AppRouteHandler } from "../../base/type";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)